workshop/simple-example.md

```diff
@@ -0,0 +1,33 @@
+<!-- Use this template when your project is not very complex and difficult. -->
+
+# Project Name TDD
+
+Author: ?
+
+## What are you building?
+
+<!-- Describe what problem are you trying to solve, what you are trying to build or what it is you are trying to do. Give an overview of the functionality. -->
+
+I really like to cook, but I'm also very disorganized. I'd like to have a better way to read my recipes.
+
+I will create a simple website for viewing my recipes.
+
+## How are you going to build it?
+
+<!-- Show how you are going to build it. You can list the steps or explain the final state. If it applies, include any diagrams, wireframes, pseudo-code, or code snippets that validate your design. Make sure to include details for any areas that are especially complex or risky. -->
+
+I'd like to use the Jamstack for this, which includes:
+
+1. Pre-rendering static html with a popular framework like Gatsby, Hugo or NextJS. See [this guide](https://jamstack.org/generators/).
```

Comment:
I'd like to see more information on these frameworks. How are you planning to decide which one to use?